src/test/java/uk/gov/pay/api/it/CreateAgreementIT.java

```diff
@@ -66,12 +70,25 @@ public void shouldReturn400WhenWhenReferenceIsNull() {
                 .body("description", is("Missing mandatory attribute: reference"));
     }
 
+    @Test
+    public void shouldReturn400WhenWhenDescriptionIsNull() {
```

Comment:
Tests for what happens if a `user_identifier` is null (not sure if we should reject it or ignore it), an empty string or too long would also make a fine addition.